### PR TITLE
Additional tests for straights

### PR DESCRIPTION
**Pull request by hkBst**
Additional tests for straights with the same tests also for straight flushes

### CODE REVIEW

**Review by petertseng — APPROVED** [review]
OK.

Now that you have added these to the Rust track, you are the person in the best position to add these tests to https://github.com/exercism/problem-specifications/blob/main/exercises/poker/canonical-data.json. **Please do so.** We do not want the Rust track to get a reputation of being selfish and refusing to share our tests with other tracks.

File: exercises/practice/poker/tests/poker.rs [hunk 1]
```diff
@@ -222,9 +229,37 @@ fn test_straight_flush_beats_four_of_a_kind() {
...
+#[test]
+#[ignore]
+fn test_aces_can_end_a_straight_flush_high() {
+    // aces can end a straight flush (10 J Q K A)
+    test(&["KC AH AS AD AC", "10C JC QC KC AC"], &["10C JC QC KC AC"])
+}
+
+#[test]
+#[ignore]
+fn test_aces_can_end_a_straight_flush_low() {
```
Comment (petertseng) [inline exercises/practice/poker/tests/poker.rs:241]: I'm not sure about "end" in this name, but the same is said of `test_aces_can_end_a_straight_low`, and I guess either end is an end

**Comment by petertseng** [comment]
Amending previous message:

Now that you have added these to the Rust track, you are the person in the best position to add these tests to https://github.com/exercism/problem-specifications/blob/main/exercises/poker/canonical-data.json. **Please do so.** We do not want the Rust track to get a reputation of being selfish and refusing to share our tests with other tracks.

(Why previous message was amended: Previously it said "it is your responsibility", but that doesn't make sense to say that, since the word "responsibility" to me means "someone expects you to do this" and "someone will levy consequences if you do this". That gives the wrong idea here)

File: exercises/practice/poker/tests/poker.rs (hunk 1 above)
Comment (hkBst) [reply exercises/practice/poker/tests/poker.rs:241]: It is called "aces can start a straight (A 2 3 4 5)" in the canonical-data.json, which is perhaps better...

**Comment by hkBst** [comment]
As requested, I've proposed these tests also for the common test specification file: https://github.com/exercism/problem-specifications/pull/2117

**Review by petertseng — APPROVED** [review]
OK, very good, thank you.

Please note that I have rewritten this main branch. Please take note of the fact that that was done (and take necessary actions) when you next update/use the main branch.

**Pull request merged**